<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="colour_check">
     <property name="text">
      <string>Colour Numbers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="length_check">
     <property name="text">
      <string>Thread Length Estimates</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="size_check">
     <property name="text">
      <string>Final Size</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="close_button">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
